src/app/Components/Components.csproj

```diff
@@ -23,15 +23,30 @@
         <SupportedOSPlatformVersion Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'android'">30.0</SupportedOSPlatformVersion>
     </PropertyGroup>
 
+    <PropertyGroup>
+        <DIPSIncludeIcons>False</DIPSIncludeIcons>
+    </PropertyGroup>
     <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net7.0-ios|AnyCPU'">
       <CreatePackage>false</CreatePackage>
+      <CodesignProvision>Automatic</CodesignProvision>
+      <CodesignKey>iPhone Developer</CodesignKey>
+    </PropertyGroup>
```

Comment:
Ska detta værmæ?

Reply:
Ja, ellers forsøker appen å signere med in house signing.